package.json

```diff
@@ -50,7 +50,7 @@
   "jest": {
     "testEnvironment": "node",
     "coverageDirectory": "./coverage/",
-    "collectCoverage": true,
+    "collectCoverage": false,
```

Comment:
Does this need to change?

Reply:
I had a look at what it does and I don't think it does. This is what it says on jests documentation:

Indicates whether the coverage information should be collected while executing the test. Because this retrofits all executed files with coverage collection statements, it may significantly slow down your tests.

So ill change it back

Reply:
However I have just noticed that it is false in author?

Reply:
It's best to remove this from this PR and maybe open a discussion / create a tech debt card to review this configuration.

Reply:
Yep ive reverted the change now and ill create a tech debt card